Week-6/pairdifference.js

```diff
@@ -0,0 +1,38 @@
+function findPair(arr, diff) {
+  let size = arr.length;
+  let pair = [];
+  let sortedArr = arr.sort();
```

Comment:
@rajeshvlega9 Good sorting is the better way to proceed in this problem.

Few more ways to solve this problem - https://www.geeksforgeeks.org/find-a-pair-with-the-given-difference/